be/src/http/action/update_config_action.cpp

```diff
@@ -130,6 +130,14 @@ Status UpdateConfigAction::update_config(const std::string& name, const std::str
             (void)StorageEngine::instance()->segment_flush_executor()->update_max_threads(
                     config::flush_thread_num_per_store * dir_cnt);
         });
+        _config_callback.emplace("lake_flush_thread_num_per_store", [&]() {
+            int threads = config::lake_flush_thread_num_per_store;
+            if (threads <= 0) {
+                threads = CpuInfo::num_cores() * 2;
```

Comment:
can be extended to if `<=0`, means liner to the the num_cores. e.g. -2 means 2*cpu_cores, -3 means 3 * cpu_cores.

Reply:
done